pkg/addons/clusterautoscaler/cluster_autoscaler_test.go

```diff
@@ -0,0 +1,49 @@
+package clusterautoscaler_test
+
+import (
+	"testing"
```

Comment:
Did you consider using ginko/gomega? That's what all other tests use...

Reply:
Sure, will re-write... 😓